site/src/pages/components/table.mdx

```diff
@@ -124,6 +124,7 @@ Data grid implements the [grid role](https://www.w3.org/TR/core-aam-1.1/#role-ma
 | Atlaskit    | [Dynamic table](https://atlaskit.atlassian.com/packages/core/dynamic-table)                                                                           | [Playground](https://atlaskit.atlassian.com/packages/core/dynamic-table/example/stateful)            | `<table>`                     | -                                                                |
 | Ant         | [Table](https://ant.design/components/table/)                                                                                                         | [Playground](https://ant.design/components/table/)                                                   | `<table>`                     | JSON object, with custom render function defined at column level |
 | Auth0       | [Table](https://styleguide.auth0.com/components/tables)                                                                                               | -                                                                                                    | `<table>`                     | -                                                                |
-| Evergreen   | [Table](https://evergreen.segment.com/components/table)                                                                                               | [Playground](https://codesandbox.io/s/evergreen-playground-component-fjbly)                          | `<div>`                       | -                                                                |
+| Evergreen   | [Table](https://evergreen.segment.com/components/table)                                                                                               | -                                                                                                    | `<table>`                     | -                                                                |
+| KoliBri     | [Table](https://public-ui.github.io/docs/components/table)                                                                                            | [Playground](https://codesandbox.io/s/evergreen-playground-component-fjbly)                          | `<div>`                       | -                                                                |
```

Comment:
it looks like you KoliBri is sandwiched between the Evergreen table link and playground